CONTRIBUTING.md

```diff
@@ -2,34 +2,35 @@
 
 ## System Requirements
 
-go 1.18+  is recommended.
+go 1.19+  is recommended.
```

Comment:
Isn't [it](https://github.com/open-feature/go-sdk-contrib/blob/main/providers/go-feature-flag/go.mod) incompatible with go 1.19? As [toolchain](https://go.dev/doc/toolchain) requiers go 1.21+.

Reply:
Yes, some providers are already on 1.21 and this here is for guidance only. Our pipelines on the other hand runs with  latest go version

Reply:
Yeah, Okay